pinot-controller/src/main/java/org/apache/pinot/controller/api/access/AccessControlUtils.java

```diff
@@ -93,4 +93,21 @@ public void validatePermission(Optional<String> tableNameOpt, AccessType accessT
           Response.Status.FORBIDDEN);
     }
   }
+
+  public void validatePermission(HttpHeaders httpHeaders, String endpointUrl,
+      AccessControl accessControl) {
+    validatePermission("you", httpHeaders, endpointUrl, accessControl);
```

Comment:
what's the rationale behind a hard-coded `you` user here?

Reply:
@apucher The original idea is to reuse the same name method. Now I have integrated the two methods in one.